temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` for debugging in committed code. Instead, use proper logging or remove the debugging statements before committing.
```
import logging

# Setup logging configuration
logging.basicConfig(level=logging.INFO)

# Example usage
logging.info('hello')
```